docs/versioned_docs/version-zenith/user-guide/ci/snippets/actions.yml

```diff
@@ -0,0 +1,20 @@
+name: dagger
+on:
+  push:
+    branches: [main]
+
+jobs:
+  dagger:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v4
+      -
+        name: Call Dagger Function
+        uses: dagger/dagger-for-github@v5
+        with:
+          version: "0.9.11"
+          verb: call
+          module: github.com/shykes/daggerverse/hello@v0.1.2
+          args: --greeting Hola --name Daggernauts --shout message
```

Comment:
this snippet is missing the function name. That'd be `hello` in this case.

Reply:
Updated